### DISCUSSION TITLE
Upgrading from 7.0.2 -> 8.0.0 -> 9.4

### DISCUSSION THREAD

**Discussion by raghbeb** [vercel/next.js#14874] [Help]
Hi,

I am trying to upgrade the code base from 7.0.2 to 8.0.0 and facing these issue. I eventually want to upgrade to 9.4.4.

1. references like these `import { ExternalLink } from '../../node_modules/some-icons/dist'` are not resolved after build.

2. 'undefined' is getting appended to the packages - GET http://localhost:8080/_next/static/development/pages/news.jsundefined net::ERR_ABORTED 404 (Not Found)

Any help would be appreciated.

**Comment by timneutkens** [comment]
> 'undefined' is getting appended to the packages - GET http://localhost:8080/_next/static/development/pages/news.jsundefined net::ERR_ABORTED 404 (Not Found)

It's likely you have changed _document in a way that did not follow the documentation 🤔
